### ISSUE TITLE
SymbolLoadingException

### ISSUE THREAD

**Issue by 00f8t** [dotnet/Silk.NET#2233]
# Summary
Silk.NET.Core.Loader.SymbolLoadingException: 'Native symbol not found (Symbol: vkGetPhysicalDeviceVideoCapabilitiesKHR)'

when trying to get Physical Device Video Capabilities

```
private void InitDecoder()
{
    KhrVideoQueue? khrVideoQueue;
    if (!vk!.TryGetDeviceExtension(instance, device, out khrVideoQueue))
    {
        throw new NotSupportedException("VK_KHR_video_queue extension not found.");
    }



    VideoDecodeH264CapabilitiesKHR videoDecodeH264CapabilitiesKHR = new()
    {
        SType = StructureType.VideoDecodeH264CapabilitiesKhr,
    };

    VideoDecodeCapabilitiesKHR videoDecodeCapabilitiesKHR = new()
    {
        SType = StructureType.VideoDecodeCapabilitiesKhr,
        PNext = &videoDecodeH264CapabilitiesKHR,
    };

    VideoCapabilitiesKHR videoCapabilitiesKHR = new()
    {
        SType = StructureType.VideoCapabilitiesKhr,
        PNext = &videoDecodeCapabilitiesKHR,
    };

    VideoDecodeH264ProfileInfoKHR videoDecodeH264ProfileInfoKHR = new()
    {
        SType = StructureType.VideoDecodeH264ProfileInfoKhr,
        StdProfileIdc = StdVideoH264ProfileIdc.Baseline,
        PictureLayout = VideoDecodeH264PictureLayoutFlagsKHR.InterlacedInterleavedLinesBitKhr,
    };

    VideoProfileInfoKHR videoProfileInfoKHR = new()
    {
        SType = StructureType.VideoProfileInfoKhr,
        PNext = &videoDecodeH264ProfileInfoKHR,
        VideoCodecOperation = VideoCodecOperationFlagsKHR.DecodeH264BitKhr,
        ChromaSubsampling = VideoChromaSubsamplingFlagsKHR.Subsampling420BitKhr,
        LumaBitDepth = VideoComponentBitDepthFlagsKHR.Depth8BitKhr,
        ChromaBitDepth = VideoComponentBitDepthFlagsKHR.Depth8BitKhr,
    };

    VideoProfileListInfoKHR videoProfileListInfoKHR = new()
    {
        SType = StructureType.VideoProfileListInfoKhr,
        ProfileCount = 1,
        PProfiles = &videoProfileInfoKHR,
    };

    if (khrVideoQueue!.GetPhysicalDeviceVideoCapabilities(physicalDevice, &videoProfileInfoKHR, out videoCapabilitiesKHR) != Result.Success)
    {
        throw new Exception("failed to get video capabilities..");
    }
}
```

![enabled extensions](https://github.com/dotnet/Silk.NET/assets/121974522/d3998a94-e80a-49d8-a4e7-54c2144f4762)

# Steps to reproduce
- Platform: Desktop
- Framework Version: .NET 8.0
- API: Vulkan
- API Version: Vulkan 1.3

**Comment by 00f8t** [comment]
a temporary solution is to get this function yourself

```
#region Additional functions
    private Result GetPhysicalDeviceVideoFormatProperties(PhysicalDevice physicalDevice, PhysicalDeviceVideoFormatInfoKHR* physicalDeviceVideoFormatInfoKHR, uint* formatPropertiesCount, VideoFormatPropertiesKHR* videoFormatPropertiesKHR)
    {
        IntPtr funcPtr = vk!.GetInstanceProcAddr(instance, "vkGetPhysicalDeviceVideoFormatPropertiesKHR");

        if (funcPtr == IntPtr.Zero)
        {
            return Result.ErrorInitializationFailed;
        }
        else
        {
            var vkGetPhysicalDeviceVideoFormatPropertiesKHR = (delegate* unmanaged<PhysicalDevice, PhysicalDeviceVideoFormatInfoKHR*, uint*, VideoFormatPropertiesKHR*, Result>)funcPtr;

            return vkGetPhysicalDeviceVideoFormatPropertiesKHR(physicalDevice, physicalDeviceVideoFormatInfoKHR, formatPropertiesCount, videoFormatPropertiesKHR);
        }
    }

    private Result GetPhysicalDeviceVideoCapabilities(PhysicalDevice physicalDevice, VideoProfileInfoKHR* videoProfileInfoKHR, VideoCapabilitiesKHR* videoCapabilitiesKHR)
    {
        IntPtr funcPtr = vk!.GetInstanceProcAddr(instance, "vkGetPhysicalDeviceVideoCapabilitiesKHR");

        if (funcPtr == IntPtr.Zero)
        {
            return Result.ErrorInitializationFailed;
        }
        else
        {
            var vkGetPhysicalDeviceVideoCapabilitiesKHR = (delegate* unmanaged<PhysicalDevice, VideoProfileInfoKHR*, VideoCapabilitiesKHR*, Result>)funcPtr;

            return vkGetPhysicalDeviceVideoCapabilitiesKHR(physicalDevice, videoProfileInfoKHR, videoCapabilitiesKHR);
        }
    }
#endregion
```

**Comment by Perksey** [comment]
To the best of my knowledge and belief this is fixed by #2237. Please try to use the latest experimental build to see if this still reproduces.
